ase/clapplugin.cc

```diff
@@ -398,10 +406,33 @@ class ClapAudioWrapper : public AudioProcessor {
     output_events_.resize (0);
   }
   void
+  update_transportinfo()
+  {
+    const auto& trans = transport();
+    const auto& tick_sig = trans.tick_sig;
+    transportinfo.flags = CLAP_TRANSPORT_HAS_TEMPO |
+                          CLAP_TRANSPORT_HAS_BEATS_TIMELINE |
+                          CLAP_TRANSPORT_HAS_SECONDS_TIMELINE |
+                          CLAP_TRANSPORT_HAS_TIME_SIGNATURE;
+    transportinfo.tempo      = tick_sig.bpm();
+    transportinfo.tsig_num   = tick_sig.beats_per_bar();
+    transportinfo.tsig_denom = tick_sig.beat_unit();
+    double beat_pos = 0, sec_pos = 0;
+    if (trans.running())
+      {
+        transportinfo.flags |= CLAP_TRANSPORT_IS_PLAYING;
+        beat_pos = trans.current_tick_d * (1.0 / TRANSPORT_PPQN);
+        sec_pos  = trans.current_seconds + trans.current_minutes * 60;
+      }
+    transportinfo.song_pos_beats = llrint (beat_pos * CLAP_BEATTIME_FACTOR);
+    transportinfo.song_pos_seconds = llrint (sec_pos * CLAP_SECTIME_FACTOR);
```

Comment:
When the transport stops (pauses), the current position indicator does not automatically jump back to 0.
I.e. I think transportinfo.song_pos_beats and transportinfo.song_pos_seconds should be unconditionally updated to the current position, regardless of running().

Reply:
I agree. I changed this so that `song_pos_*` are always updated.

Reply:
I force-pushed the new version, also rebased it to the current trunk.